pkg/apis/servicecatalog/types.go

```diff
@@ -184,9 +194,12 @@ type Instance struct {
 // InstanceSpec represents a description of an Instance.
 type InstanceSpec struct {
 	// ServiceClassName is the reference to the ServiceClass this is an
-	// instance of.  Immutable.
+	// instance of.
+	//
+	// Immutable.
```

Comment:
Why the additional newlines here? To me, it seems more unclear if the comment for `ServiceClassName` is this far removed from the declaration.